vrayGolaemSwitch/vraydirt.cpp

```diff
@@ -0,0 +1,588 @@
+
+#pragma warning( push )
```

Comment:
Hehe true. I used the vrayDirt as a starting point. 
It's been renamed in a latter commit of the same PR